sjb/config/test_cases/test_branch_origin_extended_conformance_install_system_containers.yml

```diff
@@ -95,12 +104,18 @@ extensions:
       script: |-
         docker_image_tag="$( cat ./ORIGIN_TAG )"
         docker_repository="${OS_IMAGE_PREFIX:-"openshift/origin"}"
-        docker inspect "${docker_repository}:${docker_image_tag}"
+        [[ "$(sudo atomic containers list -n --no-trunc -f runtime=runc -f image=${docker_repository}:${docker_image_tag} --json | jq '.[0].running')" == "true" ]]
```

Comment:
Sorry guys for asking wrong question and thanks @stevekuznetsov for asking the right question. @ashcrow let me check if that works for me.